### PR TITLE
Add ideas for browser extensions to develop to the readme

### PR DESCRIPTION
**Pull request by bfred-it**
Fixes https://github.com/sindresorhus/refined-github/issues/181#issuecomment-318304190

@sindresorhus like this?

As per your comment in https://github.com/sindresorhus/module-requests/issues/99, that would make it a generic extension, not related specifically to GitHub

### CODE REVIEW

**Comment by sindresorhus** [comment]
> that would make it a generic extension, not related specifically to GitHub

Sure, but its main use-case would be for GitHub, so still slightly related. Can you include it here too?

File: readme.md [hunk 1]
```diff
@@ -228,6 +228,10 @@ Once built, load it in the browser.
 - [GitHub Clean Feed](https://github.com/bfred-it/github-clean-feed) - Group news feed events by repo
 - [Do Not Merge WIP for GitHub](https://github.com/sanemat/do-not-merge-wip-for-github) - Prevents merging of incomplete PRs
 
+Want more? Here are some ideas you could develop!
+
+- [Notification Previews for GitHub](https://github.com/sindresorhus/module-requests/issues/100) - Notification Previews for GitHub
```
Comment (sindresorhus) [inline readme.md]: No point in a description if it's the same as the title.

Comment (bfred-it) [inline readme.md]: D'oh!

**Comment by sindresorhus** [comment]
And this one: https://github.com/sindresorhus/module-requests/issues/97

**Pull request merged**